patches/minecraft/net/minecraft/entity/player/PlayerEntity.java.patch

```diff
@@ -43,23 +44,31 @@
              this.func_225652_a_(false, true);
           }
        } else if (this.field_71076_b > 0) {
-@@ -258,6 +262,7 @@
+@@ -258,6 +263,7 @@
        this.func_203041_m();
        this.field_184832_bU.func_185144_a();
        this.func_213832_dB();
 +      net.minecraftforge.fml.hooks.BasicEventHooks.onPlayerPostTick(this);
     }
  
     public boolean func_226563_dT_() {
-@@ -550,6 +555,7 @@
+@@ -333,6 +339,7 @@
+    }
+ 
+    protected void func_213832_dB() {
++      if(forcedPose!=null){this.func_213301_b(forcedPose);return;}
```

Comment:
I think you can add a few newlines to make this patch more readable

Reply:
Yeah. You should aim for conciseness in patches, but not to the degree of unreadability.